lib/commit.js

```diff
@@ -11,7 +11,7 @@ var path = require('path');
 
 module.exports = function(message, opt) {
   if (!opt) opt = {};
-  if (!message || message.length === 0) {
+  if (!message || (message.length === 0 && typeof message !== 'function')) {
```

Comment:
not sure this check is actually necessary, come to think of it, since message.length will be undefined on `function`

Reply:
`!message` will determine if the message is null or not a function. I don't think we need the check for `message.length` either as `!message` will check for an empty string as well.

Reply:
Simplified this check!